torchtune/utils/_device.py

```diff
@@ -21,6 +21,19 @@
     BlockMask = torch.Tensor
 
 
+def get_world_size_and_rank() -> Tuple[int, int]:
```

Comment:
I understand the goal of this and agree with moving this function down into utils. But I also wonder whether device.py is really the best place for it (I know we already have `_get_local_rank` in here, and I know that having separate `utils/distributed.py` and `training/distributed.py` is probably confusing)

Reply:
I'm in favor of keeping it in here.